dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ReferenceConfig.java

```diff
@@ -1,535 +1,535 @@
-/*

```

Comment:
I think we should avoid unnecessary code format to show the difference clearly.

Reply:
yes agree.

Reply:
fixed